### ISSUE TITLE
Tamanho da fonte do sistema

### ISSUE THREAD

**Issue by mrjuniortec** [pop-os/cosmic-epoch#1107]
Olá caros colegas, a interface está linda e rápida pórem senti falta de mudar o tamanho da fonte do sistema direto pelo cosmic-settings. Seria ótimo se adicionar essa opção.

**Comment by oscardssmith** [comment]
I'm 2nding this request. Fractional scaling is very imperfect, so it would be really nice to be able to use 100% scaling but bump the font size up (e.g. on a 13 inch laptop with a 4k screen).

**Comment by mrjuniortec** [comment]
It's a great, thanks!

________________________________
De: Oscar Smith ***@***.***>
Enviado: sexta-feira, 1 de novembro de 2024 19:51
Para: pop-os/cosmic-epoch ***@***.***>
Cc: Junior Almeida ***@***.***>; Author ***@***.***>
Assunto: Re: [pop-os/cosmic-epoch] [Feature Request] Change system font size through COSMIC Settings (Issue #1107)


I'm 2nding this request. Fractional scaling is very imperfect, so it would be really nice to be able to use 100% scaling but bump the font size up (e.g. on a 13 inch laptop with a 4k screen).

**Comment by gui-p** [comment]
Bumping this thread, as it's the major drawback to me.

**Comment by delyan-kirov** [comment]
Does anyone know how to do this without cosmic settings?

**Comment by iHarryPotter178** [comment]
This is the only thing still missing from cosmic in my opinion..

**Comment by BartBM** [comment]
The additional scaling feature is a nice temporary solution but I would rather set the scaling to 100% and adjust the system font size myself.

**Comment by digitaltrails** [comment]
I have a couple of use cases where changing the system font would be useful.   

I have a 4k display where I often want 1cm on screen to match 1cm on paper, this so that documents and drawings can be seen at actual size, and so that architectural drawings are their stated scale on screen.

Additionally, when dealing with photos it's common to use a variety of applications, not all of which are scale aware, so at scales other than 100%, different applications render the same images at different sizes.  Plus when pixel-peeping photos, it's best that 1 pixel is 1 physical pixel.

This can all be achieved by setting the scale to 100%, but on typical 4K monitors, the UI text and icons will then be too small.  So normally I would set the UI fonts to be larger than normal, and hence my desire that COSMIC should support that option.

I'm hoping you might be able support users with these kinds of use cases. I should note, that KDE Plasma6, under Wayland, no longer supports DPI/font customization to the degree of flexibility present in Plasma5.  The best solution in Plasma6 is to buy a really big 4k display.  Supporting these use cases would be a point of difference.

**Comment by and-win** [comment]
On my ThinkPad laptop with a 1080p display, fonts look poor due to bad anti-aliasing.
On a connected 4K display, fonts appear better, but their size cannot be adjusted without scaling the entire interface.
The main issue in my case is the inability to define font size and configure anti-aliasing settings.
I believe adding the ability to customize font settings and rendering parameters would support COSMIC DE's goals.

**Comment by Dafodilrat** [comment]
Really missing this feature

**Comment by nanowuu** [comment]
I would also like to add that on another desktop like KDE with default font size 10, COSMIC apps look overlarge and the fractional scaling can't be changed without making KDE apps smaller.

**Comment by LoganSchmalz** [comment]
Related: If you were using this feature on Gnome and have switched to Cosmic on the same install, it may cause odd behavior with scaling on some applications (#2416).

You should run the following command:
```gsettings set org.gnome.desktop.interface text-scaling-factor 1.0```

**Comment by domesya** [comment]
Would love to see this feature implemented as I haven't found any other way to change the font size and it's quite large on my system, also what if one would like to use a bitmap font? is there any way to set it to use a bitmap font?

**Comment by hanhpv** [comment]
> Related: If you were using this feature on Gnome and have switched to Cosmic on the same install, it may cause odd behavior with scaling on some applications ([#2416](https://github.com/pop-os/cosmic-epoch/issues/2416)).
> 
> You should run the following command: `gsettings set org.gnome.desktop.interface text-scaling-factor 1.0`

@LoganSchmalz  You saved my life. Many thanks!

**Comment by srdoty** [comment]
I very much agree with this request. I was incredibly shocked to see that this very basic bog standard feature is missing from the Accessibility Settings. The system font size is so tiny on my 4K monitor that it is nearly unusable. I'm seriously thinking of wiping PopOS and returning to Ubuntu for precisely this reason.  

Eyestrain and headaches are simply **not** worth it!

**Comment by leviport** [comment]
@srdoty If you're interested in a potential workaround, try the scaling settings in the Displays page.

**Comment by srdoty** [comment]
@leviport Yes, I tried that. But it negates the entire purpose of investing in a 4K monitor in the first place. One naturally wants to run scaling at 100% in order to have 4 times as many windows fit on one screen, because of the incredible productivity boost. And that works great, **except** for the very tiny system fonts on the windows. This precisely why every other distro (AFAIK) includes such a capability. 

I also find fractional scaling to be less crisp than running at the native resolution.

**Comment by leviport** [comment]
I proposed it as a workaround, not a fix. This feature request remains open.

**Comment by slanden** [comment]
I'm using a laptop size portable monitor with a System76 Meerkat computer and at 100% scale everything is just right except the size of the date in the panel bar; it's just small enough to make it hard to read.

The icons are fine because they're simple and colored, and the text inside the icon dropdown menus is also fine, but the date by itself on a thin panel bar just needs to be a little larger for me personally. I need everything else scaled at 100% though for efficient use of the small screen real estate.

Being able to change the font size could make it much more readable. Great desktop environment though!

**Comment by mr-september** [comment]
+1. it's so weird having everyone scream up and down about how customizable linux is, and one of the most popularly-recommended linux distro for beginners **doesn't let you adjust font sizes**. What?

**Comment by HeavyLogic** [comment]
**Use case**

I develop websites. When I need to scale websites, I can use browser scaling. But when I'm working, I need to see my browser's content at 100% scale. So there should be some options in DE besides "scale everything". Possible solutions:

1. "Font scale" as a separate thing (that doesn't affect browser's content). It would go good together with "Spacing scale" option. Because they don't have that in KDE Plasma, you increase font size, but spacing and gaps stay the same = ugly UI.
2. "Don't scale browser's content" checkbox. This would do the trick as well. And I feel like this could be even more elegant solution.
